libstuff/SHTTPSManager.h

```diff
@@ -33,6 +33,8 @@ class SHTTPSManager : public STCPManager {
     // Close a transaction and remove it from our internal lists.
     void closeTransaction(Transaction* transaction);
 
+    int getHTTPResponseCode(const string& methodLine);
```

Comment:
Make it static so you can call it from anywhere, since it doesn't depends on any state of the object.

Reply:
👍